### DISCUSSION TITLE
Please Add Keyboard Shortcut functionality

### DISCUSSION THREAD

**Discussion by abritabroad** [lucono/karma-test-explorer#99] [General]
Title speaks for itself really. I would love to be able to focus the test explorer with the keyboard.

Example: https://github.com/hbenl/vscode-test-explorer/blob/master/README.md

**Comment by lucono** [comment]
This extension is deprecated (see the [deprecation notice](https://github.com/lucono/karma-test-explorer/tree/master?tab=readme-ov-file#deprecation-notice)).
